docs/development/managing-users/index.md

```diff
@@ -0,0 +1,43 @@
+# Managing users
+
+To manage users you will need an admin token and connection to the REANA-Server component:
+
+```console
+$ kubectl exec -ti reana-server-xxxx bash
+> read -s REANA_ADMIN_ACCESS_TOKEN
+>
+```
+
+## Create users
+
+```console
+> flask reana-users create -e john.doe@cern.ch
+User was successfully created.
+ID                                     EMAIL                 ACCESS_TOKEN
+aa37d63d-3186-45d5-aa40-5d221cb170bf   john.doe@google.com   xxxxxxxxxxxx
+```
+
+## List users
+
+```console
+> flask reana-users get
+ID                                     EMAIL                 ACCESS_TOKEN                    ACCESS_TOKEN_STATUS
+aa37d63d-3186-45d5-aa40-5d221cb170bf   john.doe@google.com   None                            pending
+b5ff2c90-d2aa-4455-805d-599990043c39   jane.doe@google.com   xxxxxxxxxxxx                    granted
+```
+
+## Grant access tokens
+
+```console
+> flask reana-users token grant --user-id aa37d63d-3186-45d5-aa40-5d221cb170bf
```

Comment:
**A.** If we have `create` and `list` or `get`, then `token grant` and `token revoke` does not read consistently... Let's use verbs everywhere, which would give `grant-token` and `revoke-token`?

**B.** We could even think of using two-word system everywhere, to have:

- `create-user`
- `list-users`
- `ban-user`
- `grant-token`
- `revoke-token`
- `list-tokens`
- ...

**C.** The same as previous but with leading namespace first:

- `user-create`
- `user-list`
- `user-ban`
- `token-grant`
- `token-list`
- `token-revoke`

**D.** The above assumed that we keep `flask reana-users ...` as the single leading CLI command for everything.  We could also create two commands, i.e. `flask reana-tokens ...` for the second namespace, to have: 

- `flask reana-users create`
- `flask reana-users list`
- `flask reana-users ban`
- `flask reana-tokens grant`
- `flask reana-tokens list`
- `flask reana-tokens revoke`

Reply:
I would go for **C** as it is the same we do for [secrets](http://docs.reana.io/reference/reana-client-cli-api/#secret-management-commands) in `reana-client`. Regarding the command, (1) it could be renamed to `reana-admin`, or (2) we could create an alias for `flask` called `reana` then the command would be `admin`, what do you think?

```console
$ flask reana-admin [user|token]-*  # (1)
$ reana admin [user|token]-*  # (2)
```

Reply:
What about `reana-admin` CLI executable, which would go nicely with `reana-client` and `reana-dev` CLI executable names?

```console
$ reana-admin user-create ...
$ reana-admin user-list ...
$ reana-admin token-grant ...
```

We could use it later for health reports tasks (https://github.com/reanahub/reana-server/issues/234):

```console
$ reana-admin status-report ...
```